Tests/taskcreatereplace.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
Could you add comments to the top of this file just explaining what it does and how it can be modified to replace other things. I don't really wanna keep this in the repository, but it could be used by other people while we are making major code changes. Also rename it to be a bit more general.